rush.json

```diff
@@ -41,7 +41,7 @@
      * The default value is false to avoid legacy compatibility issues.
      * It is strongly recommended to set strictPeerDependencies=true.
      */
-    "strictPeerDependencies": true,
+    "strictPeerDependencies": false,
```

Comment:
Lets please link this to the issue tracking our removal of peer dependencies. Once that is fixed we should re-enable this just to be safe.

Reply:
added comment